packages/nns/src/canisters/governance/request.converters.ts

```diff
@@ -228,6 +227,32 @@ const fromAction = (action: Action): RawAction => {
     };
   }
 
+  if ("OpenSnsTokenSwap" in action) {
```

Comment:
Are you using this one? Do you add all converters by default even though they are not used?

Reply:
Not yet. Yes. Because the function ends with a general throw error that needs all actions to be implemented and because we might implement a mock someday ("Mock proposals" button in neuron page)